milvus-io/milvus
# enhance: [cherry-pick]Use channel manager interface in server_test  (…

…#31621)

Tidy the following test codes

    - Remove channel in newTestServer
    - Remove newTestServerWithMeta
    - Remove newTestServer2
    - Remove testDataCoordBase
    - Use the same func for handleTTmsg and handleRPCTTmsg

See also: #31620
pr: #31621

---------
